File: business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/util/JsonOkonomiUtils.java
package no.nav.sbl.dialogarena.soknadinnsending.business.util;

import no.nav.sbl.soknadsosialhjelp.soknad.okonomi.JsonOkonomi;

public class JsonOkonomiUtils {
    public static boolean isOkonomiskeOpplysningerBekreftet(JsonOkonomi jsonOkonomi) {
        return jsonOkonomi.getOpplysninger().getBekreftelse() != null && !jsonOkonomi.getOpplysninger().getBekreftelse().isEmpty();
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/common/utils/JsonOkonomiUtils.kt
package no.nav.sosialhjelp.soknad.common.utils

import no.nav.sbl.soknadsosialhjelp.soknad.okonomi.JsonOkonomi

object JsonOkonomiUtils {
    fun isOkonomiskeOpplysningerBekreftet(jsonOkonomi: JsonOkonomi): Boolean {
        return jsonOkonomi.opplysninger.bekreftelse != null && jsonOkonomi.opplysninger.bekreftelse.isNotEmpty()
    }
}
